lib/gui/pages/main/styles/_main.scss

```diff
@@ -18,7 +18,16 @@
   margin-top: 75px;
 }
 
-.svg-icon[disabled] path {
+svg {
+  width: 100%;
+  height: 100%;
+}
+
+div.center-block > svg-icon {
+  display: inline-block;
```

Comment:
Yeah that sounds like a good idea. None of the SVGs are affected by the change in master, so I'd say it's safe.